src/modules/Builder/components/PreviewPhrasePopup/PreviewPhrasePopup.tsx

```diff
@@ -0,0 +1,72 @@
+import { useCallback } from 'react';
+import { Box } from '@mui/material';
+import { FieldArrayWithId } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import { useCustomFormContext } from 'modules/Builder/hooks';
+import { PhrasalTemplateField } from 'redux/modules';
+import { Modal, ModalProps } from 'shared/components';
+import { StyledModalWrapper } from 'shared/styles';
+
+import { StyledImage, StyledPreviewContainer } from './PreviewPhrasePopup.styles';
+
+export const PreviewPhrasePopup = ({
+  fields = [],
+  name = '',
+  imageUrl = '',
+  ...otherProps
+}: Omit<ModalProps, 'children'> & {
+  fields?: FieldArrayWithId<{ values: PhrasalTemplateField[] }, 'values', 'id'>[];
+  name?: string;
+  imageUrl?: string;
+}) => {
+  const { t } = useTranslation('app');
+  const { getValues } = useCustomFormContext();
+
+  const renderField = useCallback(
+    (_: PhrasalTemplateField, index = 0) => {
+      const fieldValue = getValues(`${name}.${index}` as string);
+
+      if (!fieldValue) {
+        return '';
+      }
+
+      const renderedName = ` ${t('phrasalTemplatePreviewPopup.responsePlaceholder', {
+        itemName: fieldValue?.itemName,
+      })} `;
+
+      switch (fieldValue.type) {
+        case 'sentence':
+          return <Box component="span">{fieldValue.text}</Box>;
+        case 'line_break':
+          return <Box component="hr" sx={{ m: 0, height: 32, border: 'none' }} />;
```

Comment:
Line breaks don't actually insert a gap… this is what the web app adds when it encounters one:
```suggestion
          return <br />;
```

Reply:
Hmm, are you sure there should be no appearance of a gap? I couldn't find explicit direction about the preview card, but I was following the [spec for the web app, which specifies that line breaks should insert a 32px gap](https://www.figma.com/design/3wcZ6bxK2QVo733ADxVjBw/SSI-2024?node-id=692-76427&t=BLw76cfZQzfaJAK6-4). 

It doesn't seem like there's a conventional/cross-browser way to style a `<br>` tag like this.

Reply:
Ah, my bad. I didn't realize this was a [bug in the web app code](https://github.com/ChildMindInstitute/mindlogger-web-refactor/blob/4b938c806d4d5dc356998dd944b845c5bb8300f5/src/entities/activity/ui/items/ActionPlan/Phrase.tsx#L46). I was using the web app as the source of truth rather than Figma. I guess I'll set a reminder to create a ticket for this next week, unless you have time to create a quick PR for the web app… 😏

Reply:
I was somewhat reluctant to open a new PR to address this, given all of @qiushihe's work that's going into refactoring the code in that exact area, but I left a [comment on the PR](https://github.com/ChildMindInstitute/mindlogger-web-refactor/pull/541/files#r1817086469) that should hopefully help explain and allow her to take care of it 🤞.